### ISSUE TITLE
Cloudflare Adapter in Directory Mode doesn't respect `base`

### ISSUE THREAD

**Issue by RichiCoder1** [withastro/astro#6471]
### What version of `astro` are you using?

v2.1.2

### Are you using an SSR adapter? If so, which one?

Cloudflare

### What package manager are you using?

npm

### What operating system are you using?

Windows

### Describe the Bug

When generating the function and SSR `_routes.json` for Cloudflare, the adapter will not respect the `base` option resulting in basic routes 404-ing.

The applies to both SSR and prerender'd routes (due to an issue both with the generated function, and the `_routes.json` not being mapped to base.

### Link to Minimal Reproducible Example

https://stackblitz.com/edit/github-4ru86g?file=astro.config.mjs

### Participation

- [ ] I am willing to submit a pull request for this issue.

**Comment by bluwy** [comment]
@RichiCoder1 is this fixed with #6473 merged?

**Comment by RichiCoder1** [comment]
@bluwy yup, was fully resolved with https://github.com/withastro/astro/pull/6550
